.github/workflows/publish.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         run: |
           VERSION=v${{steps.version_check.outputs.version}} pnpm build
           mkdir -p "$GITHUB_WORKSPACE/v${{ steps.version_check.outputs.version }}"
-          mv schema/* "$GITHUB_WORKSPACE/v${{ steps.version_check.outputs.version }}"
+          mv schemas/* "$GITHUB_WORKSPACE/v${{ steps.version_check.outputs.version }}"
```

Comment:
I think this is the only change needed here!

This has the nice benefit of preserving the old URLs in the format -

`https://theopensystemslab.github.io/digital-planning-data-schemas/<VERSION>/schema.json`

and generating new URLs in the format - 

`https://theopensystemslab.github.io/digital-planning-data-schemas/<VERSION>/schemas/<SCHEMA>.json`